Lib/test/test_gettext.py

```diff
@@ -238,6 +267,16 @@ def test_bad_minor_version(self):
             # Check that no error is thrown with a bad minor version number
             gettext.GNUTranslations(fp)
 
+    def test_corrupt_file(self):
+        with open(MOFILE_CORRUPT, 'rb') as fp:
+            with self.assertRaises(OSError) as cm:
+                gettext.GNUTranslations(fp)
+
+            exception = cm.exception
+            self.assertEqual(exception.errno, 0)
+            self.assertEqual(exception.strerror, "File is corrupt")
+            self.assertEqual(exception.filename, MOFILE_CORRUPT)
```

Comment:
```suggestion
        self.assertEqual(cm.exception.errno, 0)
        self.assertEqual(cm.exception.strerror, "File is corrupt")
        self.assertEqual(cm.exception.filename, MOFILE_CORRUPT)
```

I think this fits (I like intermediate variables in case of future improvements but I don't think this test will change :'))

Reply:
Never mind it's because the rest of the file uses that style.

Reply:
Yeah, I opted to follow the style of the other tests in this file (or at least those in close proximity) :)

Reply:
Yeah I couldn't see it in the diff